manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml

```diff
@@ -0,0 +1,49 @@
+# Configure cluster-monitoring for cluster-authentication-operator
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openshift-authentication-operator
+  namespace: openshift-authenticaion-operator
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
Does the operator need to do anything to expose metrics somehow?

Reply:
use the `WithServer` option from library-go: https://github.com/openshift/library-go/blob/dab26bb3a8dc7fccde7227194af755bbff30ce5d/pkg/controller/controllercmd/builder.go#L128